File: php/examples/getSeries.php
<?php

require_once '../src/BurningSeries.php';
$bs = new BurningSeries();

$series = $bs->getSeries();
$numSeries = count($series);

echo "We currently have {$numSeries} Series: <br />";

foreach($series as $serie) {
	echo "{$serie['series']} (ID: {$serie['id']}) <br />";
}

File: java/examples/de/jonesboard/burningseries/examples/getSeries.java
package de.jonesboard.burningseries.examples;

import de.jonesboard.burningseries.BurningSeries;
import de.jonesboard.burningseries.interfaces.SerieInterface;

public class getSeries {

	public static void main(String[] args) {
		BurningSeries bs = new BurningSeries();

		SerieInterface[] series = bs.getSeries();
		int numSeries = series.length;

		System.out.println("We currently have " + String.valueOf(numSeries) + " Series:");
		
		for(SerieInterface serie : series) {
			System.out.println(serie.getName() + " (ID: " + serie.getId() + ")");
		}
	}


}
